src/js/utils/general.js

```diff
@@ -118,15 +118,10 @@ export function getPopperOptions(attachToOptions, step) {
         (mod) => !names.includes(mod.name)
       );
 
-      popperOptions.modifiers = [
-        ...step.options.popperOptions.modifiers,
-        ...filteredModifiers
-      ];
-
-      delete step.options.popperOptions.modifiers;
+      popperOptions.modifiers = Array.from(new Set([...popperOptions.modifiers, ...filteredModifiers, ...step.options.popperOptions.modifiers]));
```

Comment:
What's the browser support on `Array.from`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Browser_compatibility